planter/Dockerfile

```diff
@@ -35,8 +35,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     unzip \
     wget \
     zip \
-    zlib1g-dev \
-    && apt-get clean && \
+    zlib1g-dev && \
+    rm -rf /var/lib/apt/lists/* \
```

Comment:
this is missing a `&&` at the end

Reply:
thanks for catching that! I'll open a follow up PR